lib/tasks/fetch_courses.rake

```diff
@@ -0,0 +1,69 @@
+require 'terms'
+require 'faraday'
+require 'openssl'
+require 'course_api'
+
+desc "rake task to fetch course term and individual course information from MaIS APIs"
+task fetch_courses: :environment do
+  current_term = Terms.current_term
+  future_term = Terms.future_terms.first
+  errors = []
+  updated = false
+
+  # Use environment variable with a default value set to a relative path
+  # to identify where this file should be
+  cert_file = Rails.root.join("config", "sul-harvester.cert")
+  key_file = Rails.root.join("config","sul-harvester.key")
+  client_cert = OpenSSL::X509::Certificate.new File.read(cert_file)
+  client_key = OpenSSL::PKey.read File.read(key_file)
+  connection = Faraday::Connection.new 'https://registry.stanford.edu', :ssl => { :client_cert => client_cert, :client_key => client_key }
```

Comment:
Maybe we could pull connection up to e.g. `CourseAPI.connection` (and share it with the `CourseAPI#request_course_api` connection stuff too)?